### PR TITLE
ci: Create a new workflow to monitor the Helm Chart version of MongoDB

### LINKED ISSUE #8148: Create pipeline to remind us to upgrade MongoDB
Since the [bitnami fiasco](https://github.com/bitnami/charts/issues/10539) happened, Keptn is unable to release new bugfix versions for older Keptn releases where a MongoDB version is used that no longer exists in Bitnami's Helm index.

To mitigate this somehow, we want to create a pipeline with the following behaviour:
- have the pipeline run on a cron schedule every 2 months
- the pipeline should check the bitnami repo for the current latest MongoDB version and compare that against the version used in master
- if the versions differ, have the pipeline create a GitHub issue to update MongoDB, and also backport that update to older versions of Keptn that don't yet have the Mongo chart directly baked in. (reference: https://github.com/keptn/keptn/issues/8151)

This can be used as a reference command: 
```
helm search repo bitnami/mongodb | grep -e "mongodb " | awk '{print $2}'
```

## DoD

- [ ] An issue is created if we get behind the latest version of mongoDB charts

### PR DESCRIPTION
**Pull request by philipp-hinteregger**
Signed-off-by: Philipp Hinteregger <philipp.hinteregger@dynatrace.com>

## This PR

- creates a new workflow that monitors the Helm Chart Version of bitnami/mongodb
- if the chart version used by Keptn differs from the latest version of bitnami, an issue gets created

Fixes #8148

## Additional Notes

Example [Issue](https://github.com/keptn/keptn/issues/8671) that gets created by this workflow

### CODE REVIEW

**Review by thisthat — DISMISSED** [review]
LGTM

File: .github/workflows/mongodb-version-check.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+jobs:
+  check-mongodb-versions:
+    name: Check MongoDB chart versions
+    runs-on: ubuntu-20.04
+    outputs:
+      CREATE_ISSUE: ${{ steps.compare_mongodb_versions.outputs.CREATE_ISSUE }}
+      BITNAMI_VERSION: ${{ steps.get_bitnami_mongodb_version.outputs.MONGODB_VERSION }}
+      KEPTN_VERSION: ${{ steps.get_keptn_mongodb_version.outputs.MONGODB_VERSION }}
+    steps:
+    - name: Check out code.
```
Comment (mowies) [inline .github/workflows/mongodb-version-check.yml:19]:
```suggestion
    - name: Check out code
```

File: .github/workflows/mongodb-version-check.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      uses: actions/checkout@v3
+
+    - name: "Get keptn mongodb version"
```
Comment (mowies) [inline .github/workflows/mongodb-version-check.yml:22]:
```suggestion
    - name: Get Keptn MongoDB version
```

File: .github/workflows/mongodb-version-check.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      id: get_keptn_mongodb_version
+      run: |
+        MONGODB_VERSION=$(cd ./installer/manifests/keptn/ && helm dependency list | grep -e "mongodb" | awk '{print $2}')
+        echo "::set-output name=MONGODB_VERSION::$MONGODB_VERSION"
+  
+    - name: "Get latest bitnami/mongodb version"
```
Comment (mowies) [inline .github/workflows/mongodb-version-check.yml:28]:
```suggestion
    - name: "Get latest Bitnami/MongoDB version"
```

File: .github/workflows/mongodb-version-check.yml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        echo "title: 'Bitnami/MongoDB version changed'" >> mongodb-version-changed.md
+        echo "labels: ['type:critical', 'area:core']" >> mongodb-version-changed.md
+        echo "---" >> mongodb-version-changed.md
+        echo "" >> mongodb-version-changed.md
+        echo "* Link to run: $GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" >> mongodb-version-changed.md
+        if [[ $GITHUB_EVENT_NAME == 'schedule' ]]; then
+          echo "* Triggered by: Scheduled build" >> mongodb-version-changed.md
+        else
+          echo "* Triggered by: @$GITHUB_ACTOR" >> mongodb-version-changed.md
+        fi
```
Comment (mowies) [inline .github/workflows/mongodb-version-check.yml:70]: you don't allow any other triggers than schedule so I don't think you need this if here

**Review by mowies — APPROVED** [review]
LGTM!

**Pull request merged**